router/router.go

```diff
@@ -1540,7 +1543,11 @@ func (rt *HandleT) commitStatusList(responseList *[]jobResponseT) {
 			if err != nil {
 				errorCode = 200 //TODO handle properly
 			}
-			sd = utilTypes.CreateStatusDetail(resp.status.JobState, 0, errorCode, string(resp.status.ErrorResponse), resp.JobT.EventPayload, eventName, eventType)
+			sampleEvent := resp.JobT.EventPayload
+			if rt.transientSources.Apply(parameters.SourceID) {
```

Comment:
[question] Not sure I understand the motivation behind this change, can you provide more context?

Reply:
yes it is the side-effect #3 of transient sources
> 3. Sample events will not be sent to the reporting service.